azure/aso/psql/package-install.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  name: psql-1
+  namespace: psql-1
+spec:
+  serviceAccountName: psql-install
+  packageRef:
+    refName: psql.azure.references.services.apps.tanzu.vmware.com
+    versionSelection:
+      constraints: 0.16.0
+  values:
+  - secretRef:
+      name: psql-1-values
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  psql-1-values
+  namespace: psql-1  
+stringData:
+  values.yml: |
```

Comment:
_nit_: I'd prefer as few as possible mandatory data values. Can we set some default in the values schema, so that the set of mandatory data values goes down?

Reply:
I think we can set a few more values to default.

For example, do we want to make the decision to make the Firewall rule a default value? 
I'm always a bit hesitant to set a "open for all" (in this case, open within Azure) as a default value.

The namespace can be `default`, and the name can be `aso-psql` or something?

The Location and `aso_controller_namespace` will be left as required fields then.

@matteo-magni

Reply:
> The namespace can be `default`

I think we don't want to set a namespace at all when the user does not provide one. Thus, all the resources would get deployed into whatever namespace the pgki is deployed into.

Reply:
The namespace is now optional:
1. if you specify it in the values file, the contents of the package will be deployed there;
2. you don't specify the namespace in the values file and the contents will be deployed to the same namespace as the package.

Reply:
> you don't specify the namespace in the values file and the contents will be deployed to the same namespace as the package.

If I read the code correctly, the namespaces defaults to `default`, which is not necessarily the same namespace as the package install. But anyway, just a nit and something to think about.